python/lsst/meas/modelfit/display/cModelDisplay.py

```diff
@@ -0,0 +1,264 @@
+import numpy as np
```

Comment:
Should have a `from __future__ import division, print_function, absolute_import` and  `__all__ = ...` before the regular imports star.